storage/storage.go

```diff
@@ -241,6 +245,26 @@ type RefreshToken struct {
 	Nonce string
 }
 
+// RefreshTokenRef is a reference object that contains metadata about refresh tokens.
+type RefreshTokenRef struct {
+	ID        string
+	CreatedAt time.Time
+	LastUsed  time.Time
```

Comment:
We might be able to get this info from the underlying refresh token. I guess we just have to figure out if we want to save database "gets".